betty/project/extension/raspberry_mint/assets/templates/section/family.html.j2

```diff
@@ -1,116 +0,0 @@
-{%- set parents = resource.resource.parents | list -%}
```

Comment:
This section file and some others are also used by our content peoy. Decouple those in a separate PR.